src/olympia/tags/cron.py

```diff
@@ -1,31 +0,0 @@
-import cronjobs
-
-from olympia.addons.models import Addon
-from olympia.tags.models import AddonTag, Tag
-
-
-@cronjobs.register
-def tag_jetpacks():
-    # A temporary solution for singling out jetpacks on AMO.  See bug 580827
-    tags = (
-        ('jetpack', {
-            '_current_version__files__jetpack_version__isnull': False
-        }),
```

Comment:
According to @andym on IRC, yes. For the loz: this cron was actually not used at all (though I also removed the tagging that was done at upload time).